naas/models/campaign.py

```diff
@@ -0,0 +1,58 @@
+from naas.models import Links, CampaignEmailTemplates
```

Comment:
This gave me an error as there is no `CampaignEmailTemplates` inside of the models.